platform/android/src/native_map_view.cpp

```diff
@@ -819,6 +820,11 @@ jni::Array<jni::Object<geojson::Feature>> NativeMapView::queryRenderedFeaturesFo
     return *convert<jni::Array<jni::Object<Feature>>, std::vector<mbgl::Feature>>(env, map->queryRenderedFeatures(box, { layers, toFilter(env, jfilter) }));
 }
 
+jni::Object<Light> NativeMapView::getLight(JNIEnv& env) {
+    mbgl::style::Light* light = map->getLight();
+    return jni::Object<Light>(createJavaLightPeer(env, *map, *light));
```

Comment:
I think this might break. You go from a pointer (nullable) to a reference without checking.

Do something like:

```
if (light) {
    return jni::Object<Light>(createJavaLightPeer(env, *map, *light));
} else {
    return jni::Object<Light>();
}
```